Makefile

```diff
@@ -189,3 +189,7 @@ mac-installer-help: ## Show installer creation help
 
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+custom-github-release:
```

Comment:
I dont think we need a new release target for that.

Reply:
okay sure